tests/test_mlflow.py

```diff
@@ -583,18 +583,23 @@ def test_log_mmm_evaluation_metrics() -> None:
     """Test logging of summary metrics to MLflow."""
     y_true = np.array([1.0, 2.0, 3.0])
     y_pred = np.array([[1.1, 2.1, 3.1]]).T
-    custom_metrics = ["r_squared", "rmse", "mae", "mape", "nrmse", "nmae"]
+    custom_metrics = ["r_squared", "rmse"]
```

Comment:
why this change?

Reply:
I thought it was calculating all of the ones despite provides custom_metrics. Only the two makes it a bit more obvious that we are not using *all*. With the longer list, I was going back and forth